raiden-dapp/CHANGELOG.md

```diff
@@ -1,24 +1,35 @@
 # Changelog
 
 ## Unreleased
+
 ### Added
+
 - [#1222] Enables state upload
 
 ### Fixed
+
+- [#1305] Redirects to home view upon shutdown
+- [#1306] Redirects to home view upon page refresh
 - [#1310] Disable state download if not connected to web3 provider.
 
+[#1305] https://github.com/raiden-network/light-client/issues/1305
+[#1306] https://github.com/raiden-network/light-client/issues/1306
 [#1222] https://github.com/raiden-network/light-client/issues/1222
```

Comment:
Thee `:` is needed otherwise it will mess with the markdown rendering like here: https://github.com/raiden-network/light-client/blob/master/raiden-dapp/CHANGELOG.md